### PR TITLE
Added all missing icons

### PR DESCRIPTION
**Pull request by wandrewkeech**
- ControlGrid44_Rotate
- SubGrid63_2Surf64
- ControlGridNStar66
- CubicNStarSurface_NStar66
- StarTrim_CubicNStar

### CODE REVIEW

**Comment by edwardvmills** [comment]
i would like to accept these, but i may end up tweaking/changing them in the future. the only thing i would want to change soon is the number of tick marks and little squares in SubGrid63_2Surf64 icon. 

is that ok with you? i see this as a very useful contribution, but i might get the urge to make things 'just so' at some point.

alternatively, we could talk about it and i could tell you my needs, etc...i'm just not how much more time you want to spend on this yourself.

**Comment by wandrewkeech** [comment]
I am very happy to take notes and make changes on your behalf. This is very fast work for me, whereas writing macros is very very slow work.

File: Resources/Icons/ControlGrid44_Rotate.svg
Comment (edwardvmills) [inline Resources/Icons/ControlGrid44_Rotate.svg:1]: this is fine, but the best way to represent this guy would be to make it look like a section of a sphere (or ellipsoid), with each edge bowed out slightly (still three line segments on each edge but puffed out)

it'd be nice to match the font and proportions from ControlGrid44. i mean you did the same height and width, but it 'feels' smaller. if you look at CubicSurface_44, you'll see i'm not shy to put content right up to the edge of the icon. there's too much info to afford padding!

File: Resources/Icons/SubGrid63_2Surf64.png
Comment (edwardvmills) [inline Resources/Icons/SubGrid63_2Surf64.png:1]: in general the black elements are pre-existing (inputs) and the colored elements are the new object

so in principle, we would need two black/grey strips to represent the input surfaces, could be  just small rectangles with a gradient. maybe two black lines that extend slightly beyond the grid? more importantly, the grid should have 5 'squares' vertically as well as horizontally.

i don't know if it'll all fit...the grid squares could be smaller. not sure if it matters, but it feels 'neat' when the symbol is mathematically correct.

**Comment by edwardvmills** [comment]
did you make this in inkscape or illustrator? the orignal set were made in inkscape. just curiosity, as long as it's compatible

**Comment by edwardvmills** [comment]
that's all i wanted to share, all the rest look great.

**Comment by wandrewkeech** [comment]
I'm a free software first kind of person, using PS or illustrator makes me absolutely crazy (so yes, I also was working in Inkscape).

Made up these two interpretations of your suggestions, please make further suggestions if needed; I'm not very confident I understood the vibe you were going for on SubGrid2Surf, but at least there are the right number of boxes.

**Comment by edwardvmills** [comment]
i merged and pulled it up in FeeCAD, looks great! thank you very much.

looks like something happened to the StarTrim icon. i had seen an icon with cutout center in the commits, but now i see a repeat of the star surface instead.

now i see that the grid44_rotate is  not showing up either. i think i need to set the image links in the scripts.

**Comment by edwardvmills** [comment]
yes, all the icon files are there, now i just need to link them properly.

**Comment by edwardvmills** [comment]
hmm...if i fix the icon links, is it going to screw up your work on the tooltips? they are on the same line as of right now.

What do you think? should i fix the links or wait?

**Pull request merged**